src/routes/resources/resources.handlers.ts

```
@@ -0,0 +1,123 @@
+  await Promise.all(
+    tagNames.map(async (tagName) => {
+      await db.insert(tags).values({ name: tagName }).onConflictDoNothing();
+      await db
+        .insert(resourceTags)
+        .values({
+          resourceId: newResource.id,
+          tagName,
+        })
+        .onConflictDoNothing();
+    })
+  );
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)